frontera/utils/misc.py

```diff
@@ -29,11 +32,11 @@ def load_object(path):
 
 
 def get_crc32(name):
-    return crc32(name) if type(name) is str else crc32(name.encode('utf-8', 'ignore'))
+    return crc32(to_bytes(name, 'utf-8', 'ignore'))
 
 
 def chunks(l, n):
-    for i in xrange(0, len(l), n):
+    for i in range(0, len(l), n):
```

Comment:
In https://pythonhosted.org/six/, it seems that in Python 2, `six.moves.range` uses `xrange`, so I'm not sure it's slower here.